packages/mm-backend/src/modules/mixin/snapshots/spec/snapshots.service.spec.ts

```diff
@@ -0,0 +1,159 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { SnapshotsService } from '../snapshots.service';
+import { MixinGateway } from '../../../../integrations/mixin.gateway';
+import { StrategyExecutor } from '../../../../common/utils/trading-dispatcher/strategy-executor';
+import { SnapshotsRepository } from '../snapshots.repository';
+import { CustomLogger } from '../../../logger/logger.service';
+import { SafeSnapshot } from '@mixin.dev/mixin-node-sdk';
+
+describe('SnapshotsService', () => {
```

Comment:
What's that?

Reply:
It's excluded to another branch: Fork-issue_17-snapshot-processing